cdap-docs/developers-manual/source/data-exploration/object-mapped-tables.rst

```diff
@@ -0,0 +1,118 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _object-mapped-table-exploration:
+
+============================================
+ObjectMappedTable Exploration
+============================================
+
+An ObjectMappedTable is a system Dataset that can write java objects to a Table
```

Comment:
An ``ObjectMappedTable`` is a system Dataset that can write Java objects to a Table